libevmasm/SemanticInformation.cpp

```diff
@@ -430,6 +432,27 @@ SemanticInformation::Effect SemanticInformation::storage(Instruction _instructio
 	}
 }
 
+SemanticInformation::Effect SemanticInformation::transientStorage(Instruction _instruction)
```

Comment:
I am assuming the same side effects as storage here ...

Reply:
Sounds about right!